simplecpp.h

```diff
@@ -69,6 +69,46 @@ namespace simplecpp {
     enum cppstd_t : std::int8_t { CPPUnknown=-1, CPP03, CPP11, CPP14, CPP17, CPP20, CPP23, CPP26 };
 
     using TokenString = std::string;
+
+#if defined(__cpp_lib_string_view) && !defined(__cpp_lib_span)
```

Comment:
what are these macros? are these standard macros? spontanously I think it might be an alternative to check some standard macro that gives us the c++ version ?